pkg/kubeapiserver/options/plugins.go

```diff
@@ -160,6 +163,7 @@ func DefaultOffAdmissionPlugins() sets.Set[string] {
 		defaultingressclass.PluginName,          // DefaultIngressClass
 		podsecurity.PluginName,                  // PodSecurity
 		validatingadmissionpolicy.PluginName,    // ValidatingAdmissionPolicy, only active when feature gate ValidatingAdmissionPolicy is enabled
+		mutatingadmissionpolicy.PluginName,      // Mutatingadmissionpolicy, only active when feature gate MutatingAdmissionpolicy is enabled
```

Comment:
nit: any reasons not to order always `mutating, validating` or the inverse?

Reply:
Also, not sure the original thinking here.  But I favor of always ordering as `mutating`, `validating`, so I'll fix this occurance.